test/functional/fixtures/regression/gh-1366/pages/index.html

```diff
@@ -0,0 +1,18 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+<title>GH-1366</title>
+</head>
+
+<body>
+<select>
+    <option value="Ford">Ford</option>
+    <option value="Vauxhall">Vauxhall</option>
+    <option value="Nissan">Nissan</option>
+</select>
+<script>
+    document.querySelector('select').addEventListener('input', function (event){window.selectValue = event.target.value});
```

Comment:
I think it's redundant. 
The main point of the issue - wrong events order. We check there that the state of the `select` element is updated before `input` event, I believe it's enough to check `value` only